### DISCUSSION TITLE
I'm getting an error when calling getToken, regarding failed unsubscription from Firebase, is this normal?

### DISCUSSION THREAD

**Discussion by ifigueroap** [capawesome-team/capacitor-firebase#435] [Q&A]
Whenever I call `FirebaseMessaging.getToken` I get this error on the console:

![captura_de_pantalla_2023-09-22_a_la_s__14 49 38](https://github.com/capawesome-team/capacitor-firebase/assets/508183/a964f8a9-deb5-49cc-aa9e-da2b8f797ab4)

I'm wondering whether this is normal, and how can we make it so it doesn't appear/happen.

Thanks

**Comment by robingenz** [comment]
This warning should not appear. Maybe this SO thread helps you: https://stackoverflow.com/q/58325574/6731412

**Reply by ifigueroap** [reply]
Thank you for your reply, I will look into it then.

**Reply by ifigueroap** [reply]
I'm not sure I get what I should do: downgrade to version 6.X?

**Reply by robingenz** [reply]
No, not that but maybe the other two answers will help. I have not had this warning so far and therefore have no experience with it.
